packages/examples/fortune-v2/job-launcher/server/src/database/migrations/1685352934954-addJobTable.ts

```diff
@@ -0,0 +1,95 @@
+import {MigrationInterface, QueryRunner, Table} from "typeorm";
+import { NS } from "../../common/constants";
+
+export class addJobTable1685352934954 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(`
+          CREATE TYPE ${NS}.job_status_enum AS ENUM (
+            'PENDING',
+            'PAID',
+            'LAUNCHED',
+            'FAILED',
+            'COMPLETED'
+          );
+        `);
+    
+        const table = new Table({
+          name: `${NS}.job`,
+          columns: [
+            {
+              name: "id",
+              type: "serial",
+              isPrimary: true
+            },
+            {
+              name: "user_id",
+              type: "int"
+            },
+            {
+              name: "chain_id",
+              type: "varchar"
+            },
+            {
+              name: "manifest_url",
+              type: "varchar",
+            },
+            {
+              name: "manifest_hash",
+              type: "varchar",
+            },
+            {
+              name: "escrow_address",
+              type: "varchar",
+              isNullable: true,
+            },
+            {
+              name: "fee",
+              type: "bigint",
+            },
+            {
+              name: "fund_amount",
+              type: "bigint",
+            },
+            {
+              name: "status",
+              type: `${NS}.job_status_enum`,
+            },
+            {
+              name: "retries_count",
+              type: "int",
+              default: 0,
+            },
+            {
+              name: "created_at",
+              type: "timestamptz",
+            },
+            {
+              name: "updated_at",
+              type: "timestamptz",
+            },
+            {
+              name: "wait_until",
+              type: "timestamptz",
+            },
+          ],
+          foreignKeys: [
+            {
+              columnNames: ["user_id"],
+              referencedColumnNames: ["id"],
+              referencedTableName: `${NS}.user`,
+              onDelete: "CASCADE",
```

Comment:
In case when we delete the user, all data related with him will be deleted. Also, we have connection One-To-Many and we can't delete the user and left jobs related with his user_id. I think we can add NO ACTION and test it how it will work in some cases.